Source/cmake/OptionsGTK.cmake

```diff
@@ -173,6 +173,7 @@ WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_PERIODIC_MEMORY_MONITOR PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_POINTER_LOCK PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SERVICE_WORKER PRIVATE ON)
 WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SHAREABLE_RESOURCE PRIVATE ON)
+WEBKIT_OPTION_DEFAULT_PORT_VALUE(ENABLE_SPEECH_SYNTHESIS PRIVATE ON)
```

Comment:
Are we confident about enabling this by default?
What about WPE?

Reply:
Let's make it off until the feature is mature. I made it on due to layout tests enabled.

Reply:
You can keep it as experimental feature, so it will be enabled for developer builds (bots, etc).